frontend-api/build.gradle

```diff
@@ -8,19 +8,23 @@ version "unspecified"
 
 dependencies {
 
+    implementation project(path: ':orchestration-shared')
+    implementation project(path: ':orchestration-shared')
```

Comment:
We don't want this dependency right? Since this is actually going to be owned by auth, we should be using the authentication version of shared